### PR TITLE
Support blending multiple values in SwitchOp operator

### PR DESCRIPTION
**Pull request by chrisgervang**
Refactored SwitchOp to handle edge cases for blending, including clamping indices beyond array length, returning exact values for integer indices, and supporting arrays and objects. Updated tests to cover new blending scenarios and edge cases.

### CODE REVIEW

File: noodles-editor/src/noodles/operators.ts [hunk 1]
```diff
@@ -1849,7 +1849,28 @@ export class SwitchOp extends Operator<SwitchOp> {
...
+        return { value: undefined }
+      }
+
+      if (values.length === 1) {
+        return { value: values[0] }
+      }
+
+      // For multiple values, we need to find which two values to interpolate between
+      // and calculate the interpolation factor
+      const clampedIndex = Math.min(index, values.length - 1)
```
Comment (akre54) [inline noodles-editor/src/noodles/operators.ts]: 👍

File: noodles-editor/src/noodles/operators.ts [hunk 2]
```diff
@@ -1849,7 +1849,28 @@ export class SwitchOp extends Operator<SwitchOp> {
     blend,
   }: ExtractProps<typeof this.inputs>): ExtractProps<typeof this.outputs> {
     if (blend) {
```
[the remark this answers is not in the document]
Comment (chrisgervang) [reply noodles-editor/src/noodles/operators.ts]: Sure, then it reads like a shortcut (which it is)
